linker/spm.jp0.lst

```diff
@@ -460,6 +460,156 @@
 800677D0:fadeSetMapChangeTransition
 8023AB64:spsndCheckBgmPlaying
 
+// new to SPM-RPG-battles
```

Comment:
This, and the other non-eu0 lsts, have a few rel addresses in the dol format instead of the rel one

Reply:
Theres other rel addresses in the non eu0 lst in the dol format, and would take a long time to convert, do I really need to?
If I have to then I will but it just seems like a lot of effort for non eu0 lsts

Reply:
I mean it can cause crashes if you don't lol, it's not just a style thing

Reply:
oops